site/content/docs/howto/cicd.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Using Pinniped for CI/CD cluster operations
+description: Using Pinniped for CI/CD cluster operations.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Use Pinniped for CI/CD
+    weight: 500
+    parent: howtos
+---
+
```

Comment:
Suggestion for a new intro paragraph to go above the existing intro:

Pinniped provides user authentication to Kubernetes clusters. It does not provide service-to-service (non-user) authentication. There are many other systems for service-to-service authentication in Kubernetes. However, there is one use case for non-human users which is supported by Pinniped. Some organizations prefer to manage access for CI/CD and other similar automation tools with user accounts drawn from the same identity provider systems as their human user accounts. In this case, there is no real difference between human and bot authentication, except for a need for bots to avoid interactive login prompts. Sometimes, humans may wish to use the same features to avoid interactive login prompts for their own scripting purposes too.

Reply:
I put something a little simpler down, take a look?

Reply:
Looks good!